README.md

```diff
@@ -96,14 +87,6 @@ To run tests, run the following command:
  ruby -I test
 ```
 
```

Comment:
**[OPTIONAL]**
- [ ] Kindly include RSpec in your linter setup to make your project configuration complete 👍

Reply:
@HAMMAS-SALEEM
thanks for your good and valuable feedback.
you mentioned very good points, I will take them into account